gc/gc-base/src/test/java/org/projectnessie/gc/base/AbstractRestGC.java

```diff
@@ -90,15 +90,15 @@ void fillExpectedContents(Branch branch, int numCommits, List<Row> expected)
             });
   }
 
-  void performGc(
+  protected void performGc(
```

Comment:
extending and overriding the same test classes with call procedure. Hence changing the access specifier.